source/manual/which-gem-to-use.html.md

```diff
@@ -1,25 +1,28 @@
 ---
 owner_slack: "#govuk-developers"
```

Comment:
Is this one useful? I didn't actually know it exists, so it seems more hopeful than practical ;-).

Reply:
I'm not sure if we want to remove this page as it might useful for beginners, but I think we could increase the review time to at least a year so it doesn't come up as often.